benchmark_runner/krkn_hub/krknhub_workloads.py

```diff
@@ -53,7 +54,7 @@ def run_workload(self):
         """
         logger.info(f'run krkn-hub: {self.__krknhub_workload}')
         pod_log_path = f'{self._run_artifacts_path}/{self.__krknhub_workload}_pod.log'
-        workload_command = f'{self.__krknhub_environment_variables}; podman run --rm --name={self.__krknhub_pod_name} --net=host --env-host=true -v /root/.kube/config:/home/krkn/.kube/config:Z quay.io/krkn-chaos/krkn-hub:{self.__krknhub_workload} > {pod_log_path}'
+        workload_command = f'{self.__krknhub_environment_variables}; chmod 775 {self.__provision_kubeconfig_path} ; podman run --rm --name={self.__krknhub_pod_name} --net=host --env-host=true -v {self.__provision_kubeconfig_path}:/home/krkn/.kube/config:Z quay.io/krkn-chaos/krkn-hub:{self.__krknhub_workload} > {pod_log_path}'
```

Comment:
Use the most restrictive permissions you can for this, which is probably 600.  The kubeconfig file isn't executable, so should not have the executable bit set; also, it almost certainly does not need group write and global read permission.

Reply:
`chmod 600` didnt work, change it to `chmod 644`

Reply:
OK, I guess something else needs to be able to access the file.